### DISCUSSION TITLE
Shouldn't product.json field "dataFolderName" be ".vscodium" instead of ".vscode-oss" ??

### DISCUSSION THREAD

**Discussion by kuolemaaa** [VSCodium/vscodium#1992] [Q&A]
As title: Shouldn't product.json field "dataFolderName" be ".vscodium" instead of ".vscode-oss" ??

Is it intended?

What kind of side effect can I experience changing that to ".vscodium" ?

**Comment by daiyam** [comment]
Yes, it's should but it's been like this since the beginning so it's hard to change it. It would break all installs...

> What kind of side effect can I experience changing that to ".vscodium" ?

None, just you will have to modify it at each install.


Maybe I could add a new patch with a `mainDataFolderName='.vscodium'` and look for it, if not there, then use the `dataFolderName`???
